src/Backend/Application.hs

```diff
@@ -38,12 +40,18 @@ backendMain = do
     backendRedisConn <- checkedConnect (appRedisConf backendSettings)
 
     runBackend Backend {..} $ do
+        -- LEGACY: flush old pre-processed Jobs queue
```

Comment:
Actually not LEGACY, since retry still relies on the ability to enqueue a pre-accepted `Job`. I have other changes staged about this, but want to merge a smaller diff first.